local_planner/src/nodes/local_planner_node.h

```diff
@@ -47,6 +47,18 @@ struct cameraData {
   bool received_;
 };
 
+enum class MAV_STATE {
+  MAV_STATE_UNINIT,
```

Comment:
Ok, if these need to match an external number, I would initialize it with 0, or  even number them all. Also just add a little comment above the enum declaration here to say where this comes from.